### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by yuripastushenko**
Fixes https://github.com/flant/k8s-image-availability-exporter/issues/13

### CODE REVIEW

File: helm/k8s-image-availability-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Configuration
+
+The following tables list the configurable parameters of the k8s-image-availability-exporter chart and their default values.
+
+### General
+| Parameter | Description | Default |
+| ----- | ----------- | ------ |
+| `k8sImageAvailabilityExporter.image.pullPolicy` | Image pull policy to use for the k8s-image-availability-exporter deployment | `IfNotPresent` |
+| `k8sImageAvailabilityExporter.image.repository` | Repository to use for the k8s-image-availability-exporter deployment | `flant/k8s-image-availability-exporter` |
+| `k8sImageAvailabilityExporter.image.tag` | Tag to use for the k8s-image-availability-exporter deployment | `v0.1.5` |
```
Comment (zuzzas) [inline helm/k8s-image-availability-exporter/README.md:44]:
```suggestion
| `k8sImageAvailabilityExporter.image.tag` | Tag to use for the k8s-image-availability-exporter deployment | `v0.1.13` |
```

File: helm/k8s-image-availability-exporter/templates/service-account.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "k8s-image-availability-exporter.fullname" . }}
```
Comment (zuzzas) [inline helm/k8s-image-availability-exporter/templates/service-account.yaml:4]: Please, add a newline at the of a file.

**Comment by yuripastushenko** [comment]
@zuzzas I made necessary changes. Please check them.

**Comment by zuzzas** [comment]
@yuripastushenko 
Thank you!

**Pull request merged**